crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -525,11 +526,11 @@ impl Default for StandardMaterial {
     }
 }
 
-impl From<LegacyColor> for StandardMaterial {
-    fn from(color: LegacyColor) -> Self {
+impl From<Color> for StandardMaterial {
+    fn from(color: Color) -> Self {
         StandardMaterial {
-            base_color: color,
-            alpha_mode: if color.a() < 1.0 {
+            base_color: color.into(),
```

Comment:
is there a reason to have a into here, even if the type is the same type as used in base_color? `Color::into<Color>()`

Reply:
Yeah, CI caught this too.